addons/account/models/account_reconcile_model.py

```diff
@@ -749,13 +811,14 @@ def _get_invoice_matching_query(self, st_lines_with_partner, excluded_ids):
         # If this reconciliation model defines a past_months_limit, we add a condition
         # to the query to only search on move lines that are younger than this limit.
         if self.past_months_limit:
-            date_limit = fields.Date.context_today(self) - relativedelta(months=self.past_months_limit)
-            query += "AND aml.date >= %(aml_date_limit)s"
+            date_limit = fields.Date.context_today(
+                self) - relativedelta(months=self.past_months_limit)
+            query += " AND aml.date >= %(aml_date_limit)s"
             params['aml_date_limit'] = date_limit
 
         # Filter out excluded account.move.line.
         if excluded_ids:
-            query += 'AND aml.id NOT IN %(excluded_aml_ids)s'
+            query += ' AND aml.id NOT IN %(excluded_aml_ids)s'
```

Comment:
Only this change is acceptable. I think you shouldn't run pre-commit/black, as it difficult to find the relevant change, so I think you should revert the rest of changes. Also, you should point to v15, as the oldest supported stable version.

Reply:
thank so much
because i run odoo ver 14, also i want fix some issue on this branch